invenio_administration/assets/semantic-ui/js/invenio_administration/api/actions.js

```diff
@@ -0,0 +1,34 @@
+import React, { Component } from "react";
+import PropTypes from "prop-types";
+import { APIRoutes } from "./routes";
+import { http } from "./config";
+
+const getResource = async (
+  apiEndpoint = undefined,
```

Comment:
I think default values as `undefined` is the default behaviour of javascript.

Reply:
yes it is, just a habit of mine to have things explicit

Reply:
No worries then :) I am just not used to see that in javascript, that's why I asked